app/react/App/scss/layout/_sidepanel.scss

```diff
@@ -20,6 +20,11 @@ $c-sidebar: $c-white;
   &.is-hidden {
     right: -100%;
   }
+
+  .only-mobile { display: inline-block; };
+  @media (min-width: 769px) {
```

Comment:
This breakpoint leaves from 769 to 1023 without the option to close the sidepanel.